libs/wingsdk/test/target-sim/queue.test.ts

```diff
@@ -179,6 +180,43 @@ test("messages are not requeued if the function fails before timeout", async ()
   ]);
 });
 
+test("messages are not requeued if the function fails after retention timeout", async () => {
```

Comment:
Cant comment on the other file 😁 but can we also add a test for the `tf-aws` and `aws-cdk` queue resources.